wp/lib/bap_wp/src/output.mli

```diff
@@ -31,9 +31,15 @@ module Constr = Constraint
     represents the registers and memory values that lead to a specific program state,
     a list of goals that have been refuted, and if specified, the paths that lead to
     the refuted goals. *)
-val print_result :
-  Z3.Solver.solver -> Z3.Solver.status -> Constr.t -> show:string list ->
-  orig:Env.t * Bap.Std.Sub.t -> modif:Env.t * Bap.Std.Sub.t -> formatter:(string option) -> unit
+val print_result
+    : ?fmt:Stdlib__format.formatter
```

Comment:
Can we write `Format.formatter` instead of `Stdlib__format.formatter`? I don't have any opinion on whether we should or should not do that. I'm just curious if there's a reason why you fully specified `Stdlib_format` instead of just `Format`. And by no means should this hold up merging this.

Reply:
There's no deeper reason for using `Stdlib__format` instead of `Format`, and using `Format` would be more consistent. I'll change this now.